kedro/template/{{ cookiecutter.repo_name }}/kedro_cli.py

```diff
@@ -131,8 +132,13 @@ def test(args):
 
 
 @cli.command()
-def install():
-    """Install project dependencies from requirements.txt."""
+@click.option("--conda", is_flag=True, multiple=False, help=CONDA_FLAG_HELP)
+def install(conda):
+    """Install project dependencies from both requirements.txt and environment.yml (optional)."""
+
+    if conda:
```

Comment:
Could we remove the `--conda` flag and instead run this only if the `src/environment.yml` exists?